src/accountsdb/index.zig

```diff
@@ -15,6 +15,8 @@ pub const AccountRef = struct {
     slot: Slot,
     location: AccountLocation,
     next_ptr: ?*AccountRef = null,
+    // NOTE: we store this information to support fast loading
+    next_index: ?u64 = 0,
```

Comment:
Would be great to explain that these fields are in sync (same for AccountReferenceHead).

Reply:
76800232